### PR TITLE
change exclude to include

### PR DESCRIPTION
**Pull request by yviansu**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -12,5 +12,5 @@
     "allowJs": true,
     "experimentalDecorators": true,
   },
-  "exclude": ["runtime-library/*", "lib/*", "**/build/*", "tools/*", "tests/*"]
+  "include": ["scripts/*", "src/*"]
```
Comment (xujuntwt95329) [inline tsconfig.json:15]: do we need to include cli?

Comment (yviansu) [reply tsconfig.json:15]: Yes, I forgot that.

**Comment by xujuntwt95329** [comment]
Have you tested the cli?

File: tsconfig.json [hunk 2]
```diff
@@ -12,5 +12,5 @@
     "allowJs": true,
     "experimentalDecorators": true,
   },
-  "exclude": ["runtime-library/*", "lib/*", "**/build/*", "tools/*", "tests/*"]
+  "include": ["scripts/*", "src/*", "cli/*"]
```
Comment (xujuntwt95329) [inline tsconfig.json:15]:
```suggestion
  "include": ["scripts/*", "src/**/*", "cli/*", "config/*"]
```

File: tsconfig.release.json [hunk 3]
```diff
@@ -12,5 +12,5 @@
       "allowJs": true,
       "experimentalDecorators": true,
     },
-    "exclude": ["runtime-library/*", "lib/*", "**/build/*", "tools/*", "tests/*"]
+    "include": ["scripts/*", "src/*", "cli/*"]
```
Comment (xujuntwt95329) [inline tsconfig.release.json:15]:
```suggestion
    "include": ["scripts/*", "src/**/*", "cli/*", "config/*"]
```

**Pull request merged**